src/Workspaces/Core/Portable/Options/SerializableOptionSet.cs

```diff
@@ -17,66 +17,64 @@
 
 namespace Microsoft.CodeAnalysis.Options
 {
-    /// <summary>
-    /// Serializable implementation of <see cref="OptionSet"/> for <see cref="Solution.Options"/>.
-    /// It contains prepopulated fetched option values for all serializable options and values, and delegates to <see cref="WorkspaceOptionSet"/> for non-serializable values.
-    /// It ensures a contract that values are immutable from this instance once observed.
-    /// </summary>
     internal sealed partial class SerializableOptionSet : OptionSet
```

Comment:
consider adding doc comment back that explains what the purpose o fthis type actualy is.